src/indexVR.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Lighbteam in A-Frame</title>
+    <meta name="description" content="Lighbeam in A-Frame">
+    <script src="js/storeChild.js"></script>
+    <script src="js/aframe/aframeHelper.js"></script>
+    <script src="https://aframe.io/releases/0.5.0/aframe.min.js"></script>
+    <script src="js/aframe/aframeForcegraph.js"></script>
```

Comment:
this file looks like it’s autogenerated from Webpack? did you write this or is it a vendor, third-party lib? in other words, how could one modify the source and regenerate it?

Reply:
@cvan [Here](https://github.com/vasturiano/aframe-forcegraph-component/blob/master/dist/aframe-forcegraph-component.js) is the main source.

- I was trying to load `forcegraph` components by passing `nodes` and `links` and it kept complaining about wrong `JSON` syntax.
- The `jsonUrl` property for `forcegraph` component worked while running on a local server, but as you pointed out earlier, when I was trying to load `indexVR.html` via `moz-extension://` it failed.
- The `forcegraph` component uses `qwest` [here](https://github.com/vasturiano/aframe-forcegraph-component/blob/master/dist/aframe-forcegraph-component.js#L134) which uses `XMLHttpRequest` and doesn't seem to work without a local server.
- Keeping MozFest in priority, the next task was to quickly look into `forcegraph`'s source code and I decided to use this hack.
- This is off-course temporary, and I will consider submitting a patch to this `forcegraph` component itself to make it work without requiring a local server for `qwest` (to fetch the `JSON` file).
- I attempted my hack directly on the `Webpack` generated file because I am running short on time (MozFest is this week) and I didn't have to worry about the `forcegraph` component failing to build etc.
- Considering this is a 1-2 week hack solution, can we leave it this way or are we violating any license rules?
cc @jonathanKingston